libsolidity/parsing/DocStringParser.cpp

```diff
@@ -164,7 +164,7 @@ DocStringParser::iter DocStringParser::parseDocTagParam(iter _pos, iter _end)
 
 DocStringParser::iter DocStringParser::parseDocTag(iter _pos, iter _end, string const& _tag)
 {
-	// LTODO: need to check for @(start of a tag) between here and the end of line
```

Comment:
The `L` is for @LefterisJP  by the way.

Reply:
Personalized TODO notes? When TODO NFTs? :P

Reply:
Ah yes. Turned out to be one of those TODOs :rofl: